addNewGame/newGame_form.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once '../vendor/autoload.php';
+require_once '../src/DB.php';
+
+use GameCollection\Models\GameModel;
+
+$name = $_POST['name'];
+$franchise = $_POST['franchise'];
+$price = $_POST['price'];
+$genre_id = $_POST['genre_id'];
+
+$errors = '';
+
+if (empty($name)) {
+    $errors .= "<p>Name is required.</p>";
+}
+
+if (empty($franchise)) {
+    $errors .= "<p>Franchise is required.</p>";
+}
+
+if (!preg_match("/^\d+\.\d{2}$/", $price)) {
+    $errors .= "<p>Price must be in the format of 10.99.</p>";
+}
+
+if ($genre_id == "Select") {
+    $errors .= "<p>Please select a genre.</p>";
+}
+
```

Comment:
You might actually be able to use this exact same validation when making your edit functionality. For that reason, you should make this into a re-usable validation function. It might also need unit tests (sozzzzzzz)